docs/exploration-and-organization/exploration.md

```diff
@@ -24,7 +24,7 @@ To learn more, see [Exploring data with Metabase's data browser](https://www.met
 
 ![A collection](./images/collection-detail.png)
 
-If your teammates are cool, they'll have pinned some important dashboards or questions within your collections; if so, those pinned items will show up as cards at the top of the collection.
+Your teammates might have pinned some important dashboards or questions within your collections; if so, those pinned items will show up as cards at the top of the collection.
```

Comment:
Rephrasing this because "If your teammates are cool" goes a bit too far into "shit-talking" territory imo